gradle-baseline-java/src/main/java/com/palantir/baseline/plugins/BaselineModuleJvmArgs.java

```diff
@@ -173,15 +175,21 @@ public void execute(Task task) {
             });
 
             project.getTasks().withType(Test.class).configureEach(test -> {
-                ModuleJvmArgsArgumentProvider provider =
-                        newModuleJvmArgsArgumentProvider(project, extension, test.getClasspath());
```

Comment:
Wow, so calling `test.getClasspath()` eagerly evaluates stuff? I'm so used to being able to do this in Gradle: `task.getSomeFileCollection().from(otherTask.getClasspath())` and it just all work lazily...